charts/opensearch/values.yaml

```diff
@@ -323,12 +323,21 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+clusterHealthCheckParams: "wait_for_status=green&timeout=1s"
```

Comment:
Hey @TheAlgo, I dont think so we need this as well `clusterHealthCheckParams:`, its not being used anywhere in templates, please correct me if i'm wrong.

Reply:
Yes @prudhvigodithi we can remove this now since we are not using authenticated probe anymore